app/src/main/java/com/github/se/gomeet/ui/mainscreens/events/ManageInvites.kt

```diff
@@ -356,13 +360,26 @@ fun UserInviteWidget(
               .testTag("UserInviteWidget"),
       horizontalArrangement = Arrangement.SpaceBetween,
       verticalAlignment = Alignment.CenterVertically) {
+        val painter: Painter =
```

Comment:
This is good because the event's actual picture now gets displayed in invitations. However, I have seen this code copy-pasted elsewhere multiple times, so at some point we should think about transferring this into a function that returns a Painter.